pyega3/data_file.py

```diff
@@ -154,7 +157,9 @@ def download_file_slice(self, file_name, start_pos, length, options=None, pbar=N
         if options is not None:
             path += '?' + urllib.parse.urlencode(options)
 
-        file_name += f'-from-{str(start_pos)}-len-{str(length)}.slice'
+        final_file_name = file_name + f'-from-{str(start_pos)}-len-{str(length)}.slice'
```

Comment:
Just nitpicking: instead of
`file_name + f'-from-{str(start_pos)}-len-{str(length)}.slice'`
I would write something like
`f'{file_name}-from-{str(start_pos)}-len-{str(length)}.slice'`.